anthos-baremetal-edge-deployment/scripts/delete-abm-gsa-keys-gcp.sh

```diff
@@ -13,6 +13,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# [START anthosbaremetal_scripts_delete_abm_gsa_keys_gcp]
+#!/bin/bash
```

Comment:
Thought: 

@ckim328, thanks for pointing out that you:
> decided to have the license placed before the region tags and #!/bin/bash

Judging from [this old blog post](https://ajurasz.github.io/2016-11-02-watch-were-you-put-your-license-headers/) from 2016, it's possible to run into issues with Docker when the shebang `#!/bin/bash` comes _after_ the license header.
The issue the author faced: `/bin/sh` was used instead of `/bin/bash`.
I'm happy to leave things as is — but I'll let the bash experts @askmeegs and @Shabirmean have the final say.

Reply:
I'll be editing those! thanks Nim!!